src/filters/enhanceContrast.ts

```diff
@@ -0,0 +1,50 @@
+import { Image } from '../Image';
+import checkProcessable from '../utils/checkProcessable';
+import { ImageColorModel } from '../utils/constants/colorModels';
+import { getOutputImage } from '../utils/getOutputImage';
+
+export interface EnhanceContrastOptions {
+  /**
+   * Image to which to output.
+   */
+  out?: Image;
+}
+
+/**
+ * Enhance the contrast of an image by spanning each channel on the range [0, image.maxValue].
+ *
+ * @param image - The image to enhance.
+ * @param options - Enhance contrast options.
+ * @returns The enhanced image.
+ */
+export function enhanceContrast(
```

Comment:
I'm not sure about the name. This is one way of enhancing contrast, but there are others and this one doesn't deserve such a generic name more, does it?

Reply:
Indeed, do you have any other idea?

Reply:
Proposition: `spanChannelToFullRange()`

Reply:
what about `autoLevel` ?

Reply:
sgtm!